sandbox/referencedStory.json

```diff
@@ -0,0 +1,163 @@
+{
+    "id": 458589,
+    "uuid": "b26212eb-d290-49c5-995b-def2d1218546",
+    "thumbnail_url": "http://avatars-cdn.prezly.test/story/1408/458589/af2b52c63ebd3189a6755063bf1080e7e067bb4feb352286c9b70e5790147f5b?t=1649086049&v=1",
+    "title": "File",
+    "subtitle": "My sub",
+    "intro": "",
+    "summary": "",
+    "culture": {
+        "code": "en",
+        "locale": "en",
+        "name": "English (Global)",
+        "native_name": "English (Global)",
+        "direction": "ltr",
+        "language_code": "en"
+    },
+    "slug": "file",
+    "format_version": 3,
+    "language": "EN",
+    "status": "draft",
+    "lifecycle_status": "draft",
+    "publication_status": "draft",
+    "visibility": "public",
+    "is_archived": false,
+    "is_finalized": false,
+    "is_published": false,
+    "is_draft": true,
+    "is_embargo": false,
+    "is_private": false,
+    "is_confidential": false,
+    "is_scheduled": false,
+    "is_analytics_available": false,
+    "is_shared_to_prpro": false,
+    "is_sharable": false,
+    "created_at": "2022-02-27T12:58:45+00:00",
+    "updated_at": "2022-04-04T15:27:29+00:00",
+    "published_at": null,
+    "scheduled_at": null,
+    "author": {
+        "id": 12417,
+        "username": "dmitrii@prezly.com",
+        "email": "dmitrii@prezly.com",
+        "name": "Dmitrii Pikulin",
+        "display_name": "Dmitrii Pikulin",
+        "first_name": "Dmitrii",
+        "last_name": "Pikulin",
+        "avatar_url": "http://avatars-cdn.prezly.test/user/12417/3f1872426416d1debb8ee4e60b897cce0400f177d777dd44bcb7fd917cbd68d8?v=1&c=602a7264775f99e6d2eafcdb5962272a03fc7172e1a71d23e05cf709cd55813f",
+        "use_case_answer": "skip",
+        "sign_in_flow": "password",
+        "created_at": "2021-11-26T12:36:22+00:00",
+        "is_terms_of_service_accepted": true,
+        "last_seen_at": "2022-04-13T09:35:10+00:00",
+        "locked_until": null
+    },
+    "newsroom": {
+        "uuid": "b60490cf-c8d4-48b8-850b-f75a881b0224",
+        "id": 1,
+        "name": "Dima test",
+        "display_name": "Dima test",
+        "subdomain": "dima-test",
```

Comment:
Could you please also update story visible metadata (like title and newsroom name) to something nicer? :) Like, use details from a public Prezly news article, or changelog.